docs/manuals/core/versioning/code_example/main_with_multiple_changes.py

```diff
@@ -0,0 +1,18 @@
+import taipy as tp
+from taipy import Config, Scope
+
+
+def my_print_algo(entry: str):
```

Comment:
may be you can rename the method does_nothing instead of print.

Reply:
I renamed it to "placeholder_algo()".
I think it is better than does_nothing.

Reply:
Well we don't ask the user to replace the algo at this stage. Maybe we can use something else like
`algo_example()`. What do you think?